src/common/base-class/construct-name-enum.ts

```diff
@@ -21,5 +21,6 @@ export enum ConstructName {
   AWSCONTENTGENAPPSYNCLAMBDA = 'C8',
   AWSRAGAPPSYNCSTEPFNKENDRA = 'C9',
   AWSWEBCRAWLER = 'C10',
-  AWSTEXTTOSQL= 'C11'
+  AWSTEXTTOSQL= 'C11',
+  AWSLLAMAINDEXDATALOADER = 'C12',
```

Comment:
please add AWSLLAMAINDEXDATALOADER in `protected static usageMetricMap : Record<string, number> ` in common/base-class.ts for generating metrics

Reply:
done